libbeat/_meta/fields.common.yml

```diff
@@ -34,3 +34,8 @@
       type: alias
       path: agent.hostname
       migration: true
+
+    - name: timeseries.instance
```

Comment:
Definitively a field we should have in ECS. 

@exekias Could you open an issue for this in the ECS repo. Their is probably also a good place for naming discussions.

Reply:
That sounds good to me, would be great to have it documented there, will open as soon as we know we will move this forward :+1:

Reply:
Do we already have an issue for it? Could not find it.

Reply:
See the TODO on this PR :)

Reply:
Ups, didn't go through the description again 🤦‍♂️